test/index.js

```diff
@@ -1,47 +1,35 @@
 'use strict'
 
 const HTTP = require('http')
-const Lab = require('lab')
 const Needle = require('needle')
 const Cache = require('../lib/cache')
 const Level = require('level')
 const Path = require('path')
+const assert = require('assert')
 
 const facilityCache = require('../lib')
 const testConfig = require('../lib/config')
 
-const EXPECTED_FACILITY = ['654321', 'existing', 'za MomConnect Existing']
-const EXPECTED_HEADERS = [
-  {name: 'value', column: 'value', type: 'java.lang.String', hidden: false, meta: false},
-  {name: 'uid', column: 'uid', type: 'java.lang.String', hidden: false, meta: false},
-  {name: 'name', column: 'name', type: 'java.lang.String', hidden: false, meta: false}
-]
 const URL = 'http://admin:district@localhost:8001/api/sqlViews/1/data.json'
```

Comment:
I think we can update this url to point to the OrganisationUnits endpoints as we are no longer make use of a sqlView.